File: ModuleFSharp/Command.fs
namespace ModuleFSharp
open FarNet
open System

[<ModuleCommand(Name = "ModuleFSharp", Prefix = "ModuleFSharp", Id = "0d526cb9-7006-4ab4-8b82-7bc5d3cb8e2b")>]
type Command1() =
    inherit ModuleCommand()

    override _.Invoke(_, e) =
        Far.Api.UI.WriteLine($"{e.Prefix}:{e.Command}", ConsoleColor.Cyan)

File: ModuleCSharp/Command.cs
using FarNet;

namespace ModuleCSharp;

[ModuleCommand(Name = "ModuleCSharp", Prefix = "ModuleCSharp", Id = "30bc4a2c-6e28-43cb-ad38-5c0ee6b7f28f")]
public class Command : ModuleCommand
{
	public override void Invoke(object sender, ModuleCommandEventArgs e)
	{
		Far.Api.UI.WriteLine($"{e.Prefix}:{e.Command}", ConsoleColor.Cyan);
	}
}
